### ISSUE TITLE
Upper eyelid location

### ISSUE THREAD

**Issue by daboe01** [obophenotype/human-phenotype-ontology#11552]
Preferred term label:
Upper eyelid location

Synonyms

    Superior palpebral location

    Upper lid location

    Upper eyelid

Definition (free text, please give PubMed ID)
An anatomical spatial pattern or localization referring to the upper eyelid, the larger and more mobile fold of skin and muscle that covers the anterior aspect of the eyeball when closed. In clinical genetics and ophthalmology, this modifier is frequently used to describe localized findings such as upper eyelid ptosis, upper eyelid colobomas, or upper eyelid retraction. [1] PMID:22418721, PMID:37777491

Parent term (use hpo.jax.org/app)
The anatomical localization of the specified phenotypic abnormality (HP:0012830)

Diseases characterized by this term ? (e.g. Orphanet or OMIM number)

    Blepharophimosis, epicanthus inversus, and ptosis (OMIM:110100) (e.g., upper eyelid ptosis)

    Fraser syndrome / Cryptophthalmos (OMIM:219000) (e.g., upper eyelid coloboma or absence)

Your nano-attribution (ORCID)
0000-0001-9042-0508

**Comment by pnrobinson** [comment]
This already existed:
[HP:6000059 Localized to upper eyelid](https://hpo.jax.org/browse/term/HP:6000059)

Adding synonyms
Thanks!
